docs/spelling_wordlist

```diff
@@ -981,3 +981,4 @@ xxxxx
 yesno
 Zealand
 Zope
+autocomplete
```

Comment:
The words in this list are alphabetized. Sort "autocomplete" instead of putting it at the end of the list.